wire-ios-sync-engine/Source/UserSession/ZMUserSession/ZMUserSession.swift

```diff
@@ -18,10 +18,20 @@
 
 import Combine
 import Foundation
+import WireAnalytics
 import WireDataModel
 import WireRequestStrategy
 import WireSystem
 
+public struct AnalyticsSessionConfiguration {
+    public let countlyKey: String
+    public let host: URL
+
+    public init(countlyKey: String, host: URL) {
+        self.countlyKey = countlyKey
+        self.host = host
+    }
+}
```

Comment:
I suggest to move this to a new file.

Reply:
Done: https://github.com/wireapp/wire-ios/commit/c62350b39e408629d1455728fba45fc120500720